memecity.engine.ui/UI/Overlay/OverlayBarItem.h

```diff
@@ -0,0 +1,17 @@
+﻿#pragma once
+#include "OverlayItem.h"
+
+class OverlayBarItem : public memecity::engine::ui::overlay::OverlayItem<int>
+{
+private:
+	int value;
```

Comment:
is dit niet een redefenition van een member variable? hij staat toch al in de base class?